### ISSUE TITLE
question about trained model you release

### ISSUE THREAD

**Issue by baaj2109** [PeterL1n/RobustVideoMatting#80]
I checked the structure of released model, for example like rvm_mobilenetv3_1280x720_s0.375_fp16.mlmodel or rvm_mobilenetv3_fp32.onnx, I found that both model didn't  have any BatchNormalization layers inside, but when I run the export code myself, the exported model does have the BatchNormalization layers. Is there any reason to remove batchnormalization layer? or how you remove batchnoremalization layers. Thank you for your share.

**Comment by PeterL1n** [comment]
BatchNorm can be fused into the Convolution layers for inference.

I think newer versions of PyTorch automatically does it. Not sure how you exported it.
